packages/account/src/Components/terms-of-use/checkbox-field.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import { Checkbox } from '@deriv/components';
+import { FieldInputProps } from 'formik';
+
+type TCheckboxFieldProps = {
+    field: FieldInputProps<boolean>;
+    id: string;
+    className: string;
+    label: string;
+};
 
 /*
  * This component is used with Formik's Field component.
  */
-const CheckboxField = ({ field: { name, value, onChange }, id, label, className, ...props }) => {
+const CheckboxField = ({ field: { name, value, onChange }, id, label, className, ...props }: TCheckboxFieldProps) => {
```

Comment:
If this component is being used with formik `Field` component, can we try wrapping this with `Field` directly and we'll be able to remove this field prop?

Reply:
Can u please elaborate?